docs/detections/prebuilt-rules/rule-details/suspicious-lsass-access-via-malseclogon.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[suspicious-lsass-access-via-malseclogon]]
+=== Suspicious LSASS Access via MalSecLogon
+
+Identifies suspicious access to LSASS handle from a call trace pointing to seclogon.dll and with a suspicious access rights value. This may indicate an attempt to leak an LSASS handle via abusing the Secondary Logon service in preparation for credential access.
```

Comment:
```suggestion
Identifies suspicious access to LSASS handle from a call trace pointing to `seclogon.dll` and with a suspicious access rights value. This may indicate an attempt to leak an LSASS handle via abusing the Secondary Logon service in preparation for credential access.
```